Automattic/wp-calypso
# Editor: Add the save status to an attribute for e2e testing

This adds a data attribute to the ground control status span. 

For e2e testing, we need to be able to ensure the post has been saved. Adding this attribute instead of using the "Saved" text so that it can be used for various locales

To Test

Open the editor and make changes to the post. When the post has been saved, inspect the "Saved" text in the Ground control and the data attribute on the span, data-e2e-status, and value should be "Saved"

cc @jeremeylduvall @hoverduck
